src/test/java/seedu/address/logic/parser/AddCommandParserTest.java

```diff
@@ -39,29 +42,29 @@ public void parse_allFieldsPresent_success() {
 
         // whitespace only preamble
         assertParseSuccess(parser, PREAMBLE_WHITESPACE + NAME_DESC_BOB
-                + ADDRESS_DESC_BOB + TAG_DESC_FRIEND, new AddCommand(expectedEatery));
+                + ADDRESS_DESC_BOB + CATEGORY_DESC + TAG_DESC_FRIEND, new AddCommand(expectedEatery));
 
         // multiple names - last name accepted
         assertParseSuccess(parser, NAME_DESC_AMY + NAME_DESC_BOB
-                + ADDRESS_DESC_BOB + TAG_DESC_FRIEND, new AddCommand(expectedEatery));
+                + ADDRESS_DESC_BOB + CATEGORY_DESC + TAG_DESC_FRIEND, new AddCommand(expectedEatery));
 
         // multiple addresses - last address accepted
         assertParseSuccess(parser, NAME_DESC_BOB + ADDRESS_DESC_AMY
-                + ADDRESS_DESC_BOB + TAG_DESC_FRIEND, new AddCommand(expectedEatery));
+                + ADDRESS_DESC_BOB + CATEGORY_DESC + TAG_DESC_FRIEND, new AddCommand(expectedEatery));
 
         // multiple tags - all accepted
-        Eatery expectedEateryMultipleTags = new EateryBuilder(BOB).withTags(VALID_TAG_FRIEND, VALID_TAG_HUSBAND)
-                .build();
+        Eatery expectedEateryMultipleTags = new EateryBuilder(BOB).withTags(VALID_TAG_FRIEND,
+                VALID_TAG_HUSBAND).build();
```

Comment:
was there a need HAHAH

Reply:
fuck i edited this class a lot la then when i delete my edits only this part stayed LMFAO